conanbuildinfo.cmake

```diff
@@ -0,0 +1,184 @@
+set(CONAN_INCLUDE_DIRS  ${CONAN_INCLUDE_DIRS})
+set(CONAN_LIB_DIRS  ${CONAN_LIB_DIRS})
+set(CONAN_BIN_DIRS  ${CONAN_BIN_DIRS})
+set(CONAN_LIBS  ${CONAN_LIBS})
```

Comment:
This whole file is unique to a specific conan configuration (os, compiler, compiler version, architecture). So this file should be removed and `.gitigniore`d.